content/en/blog/_posts/2015-04-00-Weekly-Kubernetes-Community-Hangout_11.md

```diff
@@ -8,104 +8,104 @@ Every week the Kubernetes contributing community meet virtually over Google Hang
   * paul: one variant is for env vars in addition to value field have different sources which is where you would plug in e.g. syntax you use to describe a field of an object; another source would be a source that described info about the host. have partial prototype. clean separation between what's in image vs. control plane. could use source idea for volume plugin.
-  * use case: provide info for sidecar container to contact API server
-  * use case: pass down unique identifiers or things like using UID as nique identifier
-  * clayton: for rocket or gce metadata service being available for every pod for more sophisticated things; most containers want to find endpoint of service,
+  * use case: provide info for sidecar container to contact API server.
+  * use case: pass down unique identifiers or things like using UID as unique identifier.
+  * clayton: for rocket or gce metadata service being available for every pod for more sophisticated things; most containers want to find endpoint of service.
 
 3\. preconditions/dependencies
 
 * when you create pods that talk to services, the service env vars only get populated if you create the objs in the right order. if you use dns it's less of a problem but some apps are fragile. may crash if svc they depend on is not there, may take a long time to restart. proposal to have preconds that block starting pods until objs they depend on exist.
 * infer automatically if we ask people to declare which env vars they wanted, or have dep mech at pod or rc or obj level to say this obj doesn't become active until this other thing exists.
 * can use event hook? only app owner knows their dependency or when service is ready to serve.
 * one proposal is to use pre-start hook. another is precondition probe - pre-start hook could do a probe. does anything respond when i hit this svc address or ip, then probe fails. could be implemented in pre-start hook. more useful than post-start. is part of rkt spec. has stages 0, 1, 2. hard to do in docker today, easy in rocket.
-* pre-start hook in container: how will affect readiness probe since the container might have a lock until some arbitrary condition is met if you implement with prestart hook. there has to be some compensation on when kubelet runs readiness/liveness probes if you have a hook. systemd has timeouts around the stages of process lifecycle.
-* if we go to black box model of container pre-start makes sense; if container spec becomes more descriptive of process model like systemd, then does kubelet need to know more about process model to do the right thing
+* pre-start hook in container: how will affect readiness probe since the container might have a lock until some arbitrary condition is met if you implement with prestart hook. there has to be some compensation on when kubelet runs readiness/liveness probes if you have a hook. Systemd has timeouts around the stages of process lifecycle.
```

Comment:
Systemd should be systemd, this is one kind of software...specific nouns.

Reply:
@zhangxiaoyu-zidif  Thank you for your reminding,but I think it will be better because it is the first word in the sentence.

Reply:
not worth worrying about. let's get this PR merged, and going forward try not to worry about these files per jimangel's comment ^^